src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resource/TemporaryAbsencesResourceTest.kt

```diff
@@ -120,4 +125,28 @@ class TemporaryAbsencesResourceTest : IntegrationTestBase() {
       verify(temporaryAbsenceService).getTemporaryAbsence("MDI", "A1234AA")
     }
   }
+
+  @Nested
+  inner class `Confirm arrival from temporary absence` {
+
+    @Test
+    fun `confirm arrival`() {
+      whenever(temporaryAbsenceService.temporaryAbsencesArrival(eq("G5666UK"), any())).thenReturn(InmateDetail(1L))
+      val token = getAuthorisation(roles = listOf("ROLE_VIEW_ARRIVALS"), scopes = listOf("write"))
+      val temporaryAbsencesArrival = TemporaryAbsencesArrival(
+        "NMI",
+        "ET",
+        "Comment",
+        LocalDateTime.of(2021, 11, 15, 1, 0, 0)
+      )
+
+      webTestClient
+        .post()
+        .uri("/temporary-absences/prisoner/G5666UK/temporary-absence-arrival")
+        .bodyValue(temporaryAbsencesArrival)
+        .withBearerToken(token)
+        .exchange()
+        .expectStatus().isOk
+    }
```

Comment:
can we also have tests for 

* require authentication
* requires role

Reply:
done